scripts/create_all_packages.sh

```diff
@@ -58,7 +58,6 @@ else
    sed -i -e "s/$VERSION_TO_REPLACE/$MAJOR_NUMERIC_VERSION/" tmp/manifest.json
 fi
 sed -i -e "s/$VERSION_TO_REPLACE/$VERSION/" tmp/manifest.webapp
-sed -i -e "s/$VERSION_TO_REPLACE/$VERSION/" tmp/www/index.html
```

Comment:
Shouldn't we replace the version in app.js and service-worker.js?

Reply:
In index.html we set these dynamically, hence I removed this line. I think you're right that this script could be used to set the final release values in app.js and service-worker.js.